sys/mips/82371AB.c

```diff
@@ -0,0 +1,153 @@
+/* 82371AB PCI-ISA bridge driver */
+
+#define KL_LOG KL_DEV
+#include <sys/device.h>
+#include <sys/rman.h>
+#include <sys/bus.h>
+#include <mips/malta.h>
+#include <sys/pci.h>
+#include <sys/interrupt.h>
+#include <dev/piixreg.h>
+#include <dev/isareg.h>
+#include <sys/libkern.h>
+#include <sys/devclass.h>
+
+#define IO_ISASIZE 1024
+
+#define ICU_ADDR(x) ((x) + 0)
```

Comment:
Why do we need all the defines?

Reply:
Again, we don't. No more pointless defines.